### DISCUSSION TITLE
Question Opensource

### DISCUSSION THREAD

**Discussion by Jules68Sanchez** [SOZ-Faut-etre-Sub/SOZ-FiveM-Server#25] [Q&A] [answered]
Bonsoir, j'aimerais savoir à l'heure actuelle la base du serveur est open source ainsi que le Zumble est-ce que par la suite le Launcheur, le site ainsi que le bot discord sera également open source ?

Merci d'avance.

**Comment by NariieL** [comment, marked as answer]
Hello [Jules68Sanchez](https://github.com/Jules68Sanchez) !

Le launcher ne sera strictement jamais OpenSource, car il nous protège contre toute intrusion, Dos & DDOS. Le rendre dispo' serait signer notre arrêt de mort par rapport à toutes les couches de protections que nous avons. 

Le site ( Le panel Joueur et Judicaire ) sera quant à lieu totalement OpenSource sous peu.

Des bisous. <3
